src/libraries/System.Runtime.Loader/tests/GenerateHotReloadDelta.BuildTasks/GenerateHotReloadDelta.BuildTasks.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Could / should this build task also be part of the `Microsoft.DotNet.HotReload.Utils.Generator.BuildTool` package?

Reply:
Yes. I would prefer to move it in a follow-up PR.

Reply:
Ok, actually moved the rest of the build logic in the Generator.BuiltTool package.  Also added incremental rebuild support.

@ViktorHofer mind taking one more look at this PR